src/middlewared/middlewared/plugins/vm/vms.py

```diff
@@ -74,13 +82,13 @@ async def bootloader_options(self):
         return BOOT_LOADER_OPTIONS
 
     @private
-    async def extend_vm(self, vm):
+    async def extend_vm(self, vm, context):
```

Comment:
Do you recall which was the performance impact of this change? If there was one, we have a bigger problem.